src/py/mat3ra/made/tools/modify.py

```diff
@@ -35,22 +34,33 @@ def filter_by_label(material: Material, label: Union[int, str]) -> Material:
     return new_material
 
 
-@decorator_convert_material_args_kwargs_to_structure
-def translate_to_bottom(structure: PymatgenStructure, use_conventional_cell: bool = True):
+def translate_atoms(
```

Comment:
let's have two functions maybe - `translate_by_vector` and `translate_to_z_level` to have separate - too much logic in the if below.

z_level = bottom by default